Doc/tutorial/controlflow.rst

```diff
@@ -482,9 +482,9 @@ When a final formal parameter of the form ``**name`` is present, it receives a
 dictionary (see :ref:`typesmapping`) containing all keyword arguments except for
 those corresponding to a formal parameter.  This may be combined with a formal
 parameter of the form ``*name`` (described in the next subsection) which
-receives a tuple containing the positional arguments beyond the formal parameter
-list.  (``*name`` must occur before ``**name``.) For example, if we define a
-function like this::
+receives a tuple (see :ref:`tut-tuples`) containing the positional
```

Comment:
Do you think just making `tuple` a link would be OK instead of adding the `see` part?

```suggestion
receives a :ref:`tuple <tut-tuples>` containing the positional
```